exercise-src/generate.mjs

```diff
@@ -0,0 +1,132 @@
+import fs from "node:fs/promises"
+import path from "node:path"
+
+const courseName = process.argv[2]
+const moduleName = process.argv[3]
+const projectName = process.argv[4]
```

Comment:
It’d be nice if this was explained at the top of this file and/or in the README.